core/includes/install.core.inc

```diff
@@ -1454,6 +1450,341 @@ function install_select_language_form($form, &$form_state, $files) {
   return $form;
 }
 
+/**
+ * Provide a list of available translation files.
+ *
+ * @return array
```

Comment:
Nope, this anc also returns FALSE.

But my suggestion would be to not adapt the doc block, but make the function actually return the empty array.

Reply:
See below, should always be array

Reply:
FIxed